### ISSUE TITLE
Loss -nan when I use Multi-GPUs

### ISSUE THREAD

**Issue by tensmyo** [AlexeyAB/darknet#2327]
Hi @AlexeyAB 

### Problem
- Avg Loss : 1xxx → 2.x → 1xx → nan, when I use multi-gpu. 
- but it works very well on single GPU.
- tested 2gpus,4gpus,8gpus, all avg losses were nan.

### Method / cmd 
- as you suggested , pre-trained one weights and use multi-gpu cmd to train it.
- test from without pretraining. but loss goes -nan too.

### Environment
- Dataset : VOC Dataset and cleaned the non-label images.
- Software : the latest one. 2019.01.31
- cfg setting
-- batch : 64
-- sub : 16

### Machine
- OS : Windows 10 Pro
- GPU : 2080ti * 8 
- Memory : DDR4 96GB
- SSD : 1TB
- CUDA : 10
- cudnn : 7.14 ,cuda 10, window 10
- Opencv : 3.40

Did I missed something ?

**Comment by Sauraus** [comment]
What worked for me was letting the system train the first 4000 iterations on a single GPU before switching to multi-gpu.
